docs/docs/configuration-parameters.md

```diff
@@ -44,3 +44,12 @@ It is sufficient to specify just one of the `zk` or `masters` field. If both are
 `listener` is the IP address of Mesos-DNS. In SOA replies, Mesos-DNS identifies hostname `mesos-dns.domain` as the primary nameserver for the domain. It uses this IP address in an A record for `mesos-dns.domain`. The default value is "0.0.0.0", which instructs Mesos-DNS to create an A record for every IP address associated with a network interface on the server that runs the Mesos-DNS process. 
 
 `email` is the email address of the Mesos domain name administrator. It is associated with the SOA record for the Mesos domain. The format is `mailbox-name.domain`, using a `.` instead of `@`. For example, if the email address is `root@mesos-dns.mesos`, the `email` field should be `root.mesos-dns.mesos`. The default value is `root.mesos-dns.mesos`.
+
+`dnson` is a boolean field that controls whether Mesos-DNS replies to DNS requests or not. The default value is `true`. 
```

Comment:
nit: "controls whether ... replies to DNS requests..." seems somewhat misleading. Suggest "controls whether ... listens for DNS requests..."